portiaui/app/components/browser-iframe.js

```diff
@@ -156,18 +158,30 @@ const BrowserIFrame = Ember.Component.extend({
         });
     },
 
+    cookieId: Ember.computed('spider', 'project', function() {
+        if (this.get('project') && this.get('spider')) {
+            return `cookies:${this.get('project')}/${this.get('spider')}`;
+        }
+    }),
+
     msgCookies(data) {
-        let cookies = data._data;
-        this.cookies = cookies;
-        if(window.sessionStorage){
-            window.sessionStorage.portia_cookies = JSON.stringify(cookies);
+        let cookies = data.cookies,
+            cookieId = this.get('cookieId');
+
+        if(window.localStorage && cookieId && cookies.length){
```

Comment:
we have https://github.com/funkensturm/ember-local-storage included can you refactor to use that? an example usage is in `services/ui-state.js` and `storages/*`